pds_doi_core/db/test/doi_database_test.py

```diff
@@ -0,0 +1,43 @@
+import os
+import unittest
+from pds_doi_core.db.doi_database import DOIDataBase
+from pds_doi_core.util.general_util import get_logger
```

Comment:
That is good to have this unit test.

On my side it fails when I do the project test with: pytest -s (wrong signature of a function).

The unit test run with python setup.py test do not fail. I am not 100% why or if the database test is actually run in this case.

Reply:
This test was provided because the command 'list' was missing.  I am adding this as a possible action now.  Stay tune.

Reply:
From the EPIC ticket (the parent of ticket 32) we need some criteria in the  list requests, see (https://github.com/NASA-PDS/pds-doi-service/issues/27):

This component should enable the status query of:

- DOI (1..*) - should return 0..1 DOI summary objects
- LID (1..*) - should return 0..* DOI summary objects
- LIDVID (1..*) - should return 0..1 DOI summary objects
- Submitter (1..*) - should return 0..* DOI summary objects
- Discipline Node Abbreviation (1..*) - should return 0..* DOI summary objects
- Submission datetime range (start, end) (ISO-8601 format) - should return 0..* DOI summary objects

So that list command and the run method 'run' of this action should take these arguments. All of them are optional, most of them are lists.

Reply:
The criteria has been added to the query builder with the proper 'WHERE' clause depending on if it is a time related field or regular field.